src/schema/v2/me/__tests__/index.test.ts

```diff
@@ -495,11 +506,44 @@ describe("me/index", () => {
           email: true,
         })
 
-      const data = await runAuthenticatedQuery(query, {
+      const filterArtworksLoader = jest.fn().mockReturnValue(
+        Promise.resolve({
+          hits: [
+            {
+              title: "Soup can",
+            },
+          ],
+          aggregations: {
+            total: {
+              value: 1,
+            },
+          },
+        })
+      )
+
+      const context = {
         meLoader,
         meAlertLoader,
         artistLoader: () => Promise.resolve({ name: "Andy Warhol" }),
-      })
+        authenticatedLoaders: {
+          filterArtworksLoader,
+        },
+        unauthenticatedLoaders: {
```

Comment:
not sure why I need to mock both authenticated and unauthenticated loaders. I think that only unauthenticated loader should be called according to [this condition](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/filterArtworksConnection.ts#L542). But if I don't specify authenticatedLoader then I get this error:

```
Cannot read properties of undefined (reading 'filterArtworksLoader')

      at defaultMergedResolver (node_modules/graphql-tools/src/stitching/defaultMergedResolver.ts:18:24)
```

Reply:
I think b/c of the nested destructuring happening in the resolver - https://github.com/artsy/metaphysics/blob/03369e261e27940174da51793c140f5532524a89/src/schema/v2/filterArtworksConnection.ts#L483 - you need to provide it (even though just the unauthenticated one is being invoked).